src/p11_child/p11_child_openssl.c

```diff
@@ -1847,16 +1848,19 @@ errno_t do_card(TALLOC_CTX *mem_ctx, struct p11_ctx *p11_ctx,
          * the requested ones for pre-auth. For authentication all attributes
          * must be given and match, for pre-auth only the given ones must
```

Comment:
`For authentication all attributes must be given and match`  --  with exception of label?

Reply:
fixed